src/main/java/org/stellar/sdk/requests/ResponseHandler.java

```diff
@@ -1,5 +1,6 @@
 package org.stellar.sdk.requests;
 
+import com.google.common.base.Optional;
```

Comment:
this looks like an un-used import now, does your IDE highlight/warn? if using Intellij, can do 'optimize imports', will re-order and remove unused.